pybigquery/api.py

```diff
@@ -0,0 +1,22 @@
+"""Integration with BigQuery API."""
+
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+from google.cloud.bigquery import Client, QueryJobConfig
+
+
+class ApiClient(object):
+    def __init__(self, credentials_path=None):
```

Comment:
I'd prefer if this took a `google-auth` library credentials object. That way it can support other kinds of credentials such as Compute Engine credentials or user credentials.

Reply:
But this is okay to me if Superset expects a path here.

Reply:
Yes we use path in superset